### DISCUSSION TITLE
Pull request acceptatie & DOD

### DISCUSSION THREAD

**Discussion by sanderdekroon** [OpenWebconcept/Afspraken#5] [General]
Welke regels gaan we opstellen voor het accepteren van pull requests (PRs)? 

Het lijkt me niet wenselijk dat een PR van partij A ook direct geaccepteerd (en dus gemerged) wordt door zichzelf. Ik wil voorstellen dat een andere partij (marktpartij en/of gemeente) de PR ook moet goedkeuren.

Daarnaast wil ik ook de discussie starten waaraan een PR moet voldoen. Een paar punten die me te binnen schieten:
- Test coverage? Zo ja, welke test typen en via welk framework?
- Code style (PSR-2/PSR-12/WordPress?)
- Changelog bijgewerkt

Zie ook de [CONTRIBUTING.md](https://github.com/OpenWebconcept/.github/blob/main/CONTRIBUTING.md) in de .github repo.

**Comment by MWest2020** [comment]
Peer reviews werken hier inderdaad goed. Misschien dat er personen binnen de partijen hiervoor aangewezen kunnen worden en er zo elkaar werk bekeken en besproken wordt? Constructieve feedback is altijd mooi meegenomen.

Ik zou duidelijke `commit messages` toevoegen. Verder of her mogelijke security issues of kan leveren of er documentatie ergens aangepast moet worden.
